src/base/utils/os.cpp

```diff
@@ -100,9 +106,11 @@ void Utils::OS::shutdownComputer([[maybe_unused]] const ShutdownDialogAction &ac
 
 #elif defined(Q_OS_MACOS)
     AEEventID EventToSend;
```

Comment:
It should be initialized to something:
```suggestion
    AEEventID EventToSend {};
```
It is not related to your PR but it would be nice if you could help here.